src/helper.js

```diff
@@ -0,0 +1,10 @@
+const helper = {
+  getUserData: (userData, id) => {
+    return userData.filter(data => data.userID === id);
+  },
+}
+
+
+if (typeof module !== "undefined") {
+  module.exports = helper;
+}
```

Comment:
This probably isn't doing anything - your module shouldn't ever be undefined because you always define it before exporting it. Overall though you have the right idea for using a helper file. You can create variables or functions and export them. You can either require them in the files that you want to use the helper functions in (like you have in your hydration test) or load them in via your script file.